docs/pages/access-controls/guides/headless.mdx

```diff
@@ -25,17 +25,17 @@ For example:
 
 ## Prerequisites
 
-- A v12.2+ Teleport cluster with WebAuthn configured.
+- A Teleport cluster with WebAuthn configured.
   See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
 - WebAuthn hardware device, such as YubiKey.
-- Machines for Headless WebAuthn activities have [Linux](../../installation.mdx#linux), [macOS](../../installation.mdx#macos) or [Windows](../../installation.mdx#windows-tsh-and-tctl-clients-only) `tsh` binary v12.2+ installed.
+- Machines for Headless WebAuthn activities have [Linux](../../installation.mdx#linux), [macOS](../../installation.mdx#macos) or [Windows](../../installation.mdx#windows-tsh-client-only) `tsh` binary installed.
```

Comment:
This looks like a regression (broken link).

Reply:
It looks like `## Windows (tsh and tctl clients only)` is the correct header